cmd/server/pactasrv/pactasrv.go

```diff
@@ -100,3 +103,11 @@ func dereference[T any](ts []*T, e error) ([]T, error) {
 	}
 	return result, nil
 }
+
+func getUserID(ctx context.Context) (pacta.UserID, error) {
+	userID, err := jwtauth.UserIDFromContext(ctx)
```

Comment:
CI seems to be passing, but I have no idea how this code works, or the `jwtauth.WithUserId` elsewhere. These functions don't appear to exist in [the `jwtauth` package](https://pkg.go.dev/github.com/go-chi/jwtauth/v5#section-readme)

Reply:
Fixed